### DISCUSSION TITLE
管理システム

### DISCUSSION THREAD

**Discussion by anminmakura** [TUSB/TheUnusualSkyBlock#97] [各種システムまとめ]
主に全体の管理システムの仕様ついてのまとめ。

**Comment by anminmakura** [comment]
負荷軽減機構の作成。
MOBやプレイヤーのスキルによるパーティクルの抑制を1、1/2、1/3、1/4と量を削減する。

**Comment by anminmakura** [comment]
難易度によっていろいろ変わるとおもしろそう(実装するかは要検討)
・通常だとアスレで跳躍使えなくて、ピクニックだと使える
・「生きてる」シリーズの種類がかわる(ダイヤ、鉄、石みたいに)

・ピクニック、カジュアル、エキスパートで敵の出現Lvを上げたい。

**Reply by NePonpon** [reply]
以降こちらへ引き継ぎ
- https://github.com/TUSB/TheUnusualSkyBlock/discussions/131
